user_guide_src/source/outgoing/localization.rst

```diff
@@ -254,3 +254,43 @@ project:
 
 The translated messages will be automatically picked
 up because the translations folders get mapped appropriately.
+
+.. _generating-translation-files-via-command:
+
+Generating Translation Files via Command
+========================================
+
+.. versionadded:: 4.5.0
+
+You can automatically generate and update translation files in your **app** folder. The command will search for the use of the ``lang()`` function, combine the current translation keys in **app/Language** by defining the locale ``defaultLocale`` from ``Config\App``.
+After the operation, you need to translate the language keys yourself.
+The command is able to recognize nested keys normally ``File.array.nested.text``.
+Previously saved keys do not change.
+
+.. code-block:: console
+
+    php spark lang:find
+
+.. literalinclude:: localization/019.php
+
+
```

Comment:
Tell me how to write better: 
After executing the command, it is preferable to format the language files. For example, `vendor/bin/php-cs-fixer fix ./app/Language` if php-cs-fixer is installed.

Reply:
How about this?

> The language files generated will most likely not conform to your coding standards.
> It is recommended to format them. For example, run `vendor/bin/php-cs-fixer fix ./app/Language` if php-cs-fixer is installed.

Reply:
Ok